### DISCUSSION TITLE
ecash (XEC) reject blocks with bad-cb-minerfund error

### DISCUSSION THREAD

**Discussion by btclinux** [coinfoundry/miningcore#1541] [Q&A]
Hi! I see in coins.json for ecash https://github.com/oliverw/miningcore/blob/master/src/Miningcore/coins.json#L908
```"hasMinerFund": true```

All blocks rejected with:
```
ERROR: ConnectTip: ConnectBlock 0000000000000000097c03ca368bc4efbc271a1ba308877befdf69d62cdb3a5c failed, bad-cb-minerfund
```
Should I add something special in pool config?

ecash node config:
```
rpcuser=rpcuser
rpcpassword=rpcpassword
prune=550
deprecatedrpc=accounts
usecashaddr=0
zmqpubhashblock=tcp://127.0.0.1:15101
```

**Comment by oliverw** [comment]
Looks like XEC is currently broken ;(
